### ISSUE TITLE
Svg.Text with Korean character spacing problem after upgrading 6.2.2

### ISSUE THREAD

**Issue by HerbertLim** [react-native-community/react-native-svg#719]
I used Expo SDK v25.0.0, which includes react-native-svg 5.5.1.
I just upgrade Expo to v26.0.0, which includes react-native-svg 6.2.2.

In case of Android, texts in Korean  rendered with Svg.Text (just below the color horizontal lines) is good as shown below. In 5.5.1, it looked like this both at Android and iOS.
![rn-svg-android](https://user-images.githubusercontent.com/22055241/41714552-a4462050-758b-11e8-8919-add3e68113d2.png)

However, after updating to 6.2.2, in case of iOS, Korean texts, which I circled with red lines, are some what broken: spaces between Korean characters are not even.
![rn-svg-ios](https://user-images.githubusercontent.com/22055241/41714558-a68c26f2-758b-11e8-8d3d-c2fc377e6ad0.png)

It is likely that alphabet is not affected.

**Comment by liuqiang1357** [comment]
I have the same issue with Chinese character.

**Comment by msand** [comment]
Could you make a minimal reproduction of it? Otherwise it's a bit hard to investigate. I would recommend debugging and checking here: https://github.com/react-native-community/react-native-svg/blob/master/android/src/main/java/com/horcrux/svg/TSpanShadowNode.java#L265

**Comment by HerbertLim** [comment]
Dear @msand, 
I created a very simple expo project to reproduce this issue and uploaded it to following git repo.
https://github.com/HerbertLim/expo-28-svg-korean-issue.git

I was able to reproduce this issue.
At Android phone, both Korean and English text is displayed as expected.
![2018-06-24 18 50 53](https://user-images.githubusercontent.com/22055241/41817961-07c3d9fa-77e0-11e8-9cea-1b4ea5a683a7.jpg)

However, at iOS, both Korean is not displayed as expected. Even mix of Korean and English also has problem.
![2018-06-24 18 50 37](https://user-images.githubusercontent.com/22055241/41817968-4c27de52-77e0-11e8-8cdd-79d592a5640f.png)
